pkg/api/pgd_client.go

```diff
@@ -131,6 +135,40 @@ func (c PGDClient) CalculateWitnessGroupParams(ctx context.Context, projectId st
 	return &response.Data, err
 }
 
+func (c PGDClient) GetServiceAccountIds(ctx context.Context, projectID string, cspID string, regionID string) (*models.ServiceAccountIds, error) {
```

Comment:
can we not reuse these functions from the cluster client ?

Reply:
Yes good point